instaloader/structures.py

```diff
@@ -50,7 +50,26 @@ class PostComment(NamedTuple):
     owner: 'Profile'
     likes_count: int
     answers: Iterator[PostCommentAnswer]
+    context: InstaloaderContext
+    shortcode: str
 
+    @property
+    def likes(self) -> Iterator['Profile']:
+        """
+        Iterate over all likes of a comment. A :class:`Profile` instance of each like is yielded.
+
+        .. versionchanged:: 4.10
+        """
+        if self.likes_count != 0:
+            return NodeIterator(
+                self.context,
+                '5f0b1f6281e72053cbc07909c8d154ae',
+                lambda d: d['data']['comment']['edge_liked_by'],
+                lambda n: Profile(self.context, n),
+                {'comment_id': self.id},
+                'https://www.instagram.com/p/{0}/'.format(self.shortcode),
+            )
+        return iter([])
```

Comment:
I think it would be easier to make this function return `Iterable['Profile']` instead and then `return []` here.